experiment.py

```diff
@@ -76,8 +77,7 @@ def __init__(self, entry):
         for cell in self.cells.values():
             for label in labels:
                 assert label in cell.labels
-            for cre in cre_types:
-                assert cre in cell.labels
+            assert cell.cre_type in cre_types
```

Comment:
It originally made sure that, if a cre type is present for an experiment, all cells are listed as either positive or negative for that cre type. This would protect against mistakes in the summary file; I think it's still relevant.